tests/E2E Tests/WebAppUiTests/UiTestHelpers.cs

```diff
@@ -179,6 +180,13 @@ public static Process StartProcessLocally(
             }
             else
             {
+                // Log the output and error streams
+                process.OutputDataReceived += (sender, e) => Console.WriteLine(e.Data);
+                process.ErrorDataReceived += (sender, e) => Console.Error.WriteLine(e.Data);
+
+                process.BeginOutputReadLine();
+                process.BeginErrorReadLine();

```

Comment:
how does the test display them? 
Is the output of the test captured? should we use `ITestOutputHelper .WriteLine()`?
https://xunit.net/docs/capturing-output

Reply:
Thanks for bring this up - updated to use ITestOutputHelper!